src/api/customer/customer-route.ts

```diff
@@ -1,8 +1,14 @@
 import { Router } from 'express';
+import { celebrate, Segments, Joi } from 'celebrate';
```

Comment:
Estes módulos foram importados, mas não foram usados neste arquivo.

```suggestion
import { celebrate, Segments, Joi } from 'celebrate';
```